packages/squidex/schema/crn/schemas/working-groups.json

```
@@ -14,6 +14,23 @@
     "fieldsInReferences": [],
     "fieldsInLists": [],
     "fields": [
+      {
+        "name": "complete",
+        "properties": {
+          "isRequired": false,
+          "isRequiredOnPublish": false,
+          "isHalfWidth": true,
+          "fieldType": "Boolean",
+          "editor": "Toggle",
+          "inlineEditable": true,
+          "defaultValue": false,
+          "label": "This group is complete"
```

I don't know if it will occupy a lot of space in the ui, but  maybe we could change to:
```suggestion
          "label": "This working group is complete"
```